include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -791,6 +791,27 @@ def TT_ScanReturnOp: TT_Op<"scan.return",
     let assemblyFormat = "$result attr-dict `:` type($result)";
 }
 
+//
+// Map Elementwise op
+//
+def TT_MapElementwiseOp: TT_Op<"map_elementwise", [Elementwise,
+                                                   SameOperandsAndResultEncoding,
+                                                   SameOperandsAndResultShape,
+                                                   RecursiveMemoryEffects]> {
+    let summary = "Map a scalar subregion over a tensor";
+    let arguments = (ins Variadic<TT_Tensor>:$srcs);
+    let results = (outs Variadic<TT_Tensor>:$result);
```

Comment:
would it be possible to add a `pack` option similar to `inline_asm_elementwise`? This would be useful for vectorization with different lengths